learningReact/demo_react/src/App.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+// import logo from './logo.svg';
+import './App.css';
+import Header from './components/Header';
+import Content from './components/Content';
+import Footer from './components/Footer';
+// import DemoProps from './Props.DemoProps'
+const student = {id:  1, name:"nhung",age:20}
+const menu = { title:'Welcome to React JS', name:["Home", "About", "Contact","Video","Language"]}
+function App() {
+  
+  return (
+    <>
+    <Header data= {menu}/>
```

Comment:
let double check your code style, it still not good for now. Add more space here.